bindings/python/CONTRIBUTING.md

```diff
@@ -0,0 +1,388 @@
+likelihood for getting review sooner shoots up.
+
+**Self Review Guidelines to follow**
+
+-   If the PR is too large, split it if possible.
+
+    - Use 250 LoC (excluding test data and config changes) as a
+            rule-of-thumb.
+
+     - Moving and changing code should be in separate PRs or commits.
+
+        -   Moving: Taking large code blobs and transplanting
+            them to another file. There\'s generally no (or very
+            little) actual code changed other than a cut and
+            paste. It can even be extended to large deletions.
+        -   Changing: Adding code changes (be that refactors or
+            functionality additions/subtractions).
+        -   These two, when mixed, can muddy understanding and
+            sometimes make it harder for reviewers to keep track
+            of things.
+
+-   Prefer explaining with code comments instead of PR comments.
+
+**Provide background**
+
+-   The PR description and linked tickets should answer the "what" and
+    "why" of the change. The code change explains the "how".
+
+**Follow the Template**
+
+-   Please do not deviate from the template we make; it is there for a
+    lot of reasons. If it is a one line fix, we still need to have
+    context on what and why it is needed.
+
+-   If making a versioning change, please let that be known. See examples below:
+
+    -   `versionadded:: 3.11`
+    -   `versionchanged:: 3.5`
+
+**Pull Request Template Breakdown**
+
+-  **Github PR Title**
+
+    -   The PR Title format should always be
+            `[JIRA-ID] : Jira Title or Blurb Summary`.
+
+-  **JIRA LINK**
+
+-   Convenient link to the associated JIRA ticket.
+
+-   **Summary**
+
+     -   Small blurb on why this is needed. The JIRA task should have
+            the more in-depth description, but this should still, at a
+            high level, give anyone looking an understanding of why the
+            PR has been checked in.
+
+-    **Changes in this PR**
+
+     -   The explicit code changes that this PR is introducing. This
+            should be more specific than just the task name. (Unless the
+            task name is very clear).
+
+-   **Test Plan**
+
+    -   Everything needs a test description. Describe what you did
+            to validate your changes actually worked; if you did
+            nothing, then document you did not test it. Aim to make
+            these steps reproducible by other engineers, specifically
+            with your primary reviewer in mind.
+
+-   **Screenshots**
+
+    -   Any images that provide more context to the PR. Usually,
+            these just coincide with the test plan.
+
+-   **Callouts or follow-up items**
+
+    -   This is a good place for identifying "to-dos" that you've
+            placed in the code (Must have an accompanying JIRA Ticket).
+    -   Potential bugs that you are unsure how to test in the code.
+    -   Opinions you want to receive about your code.
+
+# Contributing to PyMongo
```

Comment:
Everything from here until `# Installing from source` is a duplicate above. i.e. it was pasted twice.

Reply:
Removed